types.bal

```diff
@@ -118,12 +118,23 @@ public type ExperimentListResponse record {|
 # + experiment - The full experiment data from the database
 # + return - The api response
 public isolated function mapToExperimentResponse(database:ExperimentFull experiment) returns ExperimentResponse {
-    return {
-        '\@self: string `${serverHost}/experiments/${experiment.experimentId}`,
-        experimentId: experiment.experimentId,
-        name: experiment.name,
-        description: experiment.description
-    };
+    if experiment?.templateId != () {
```

Comment:
If possible rewrite this to initialize the object with all required values first and then (if applicable) add the template id to that object before returning to get rid of duplicated code.